### PR TITLE
WIP: Fixing #139 by using conda build MetaData object to load meta.yaml info

### LINKED ISSUE #139: Feedstock Repo script failing
Running into an exception for the last 2 days in the `feedstocks_repo` build. This appears to be far enough into the script that none of us noticed. This can be seen in the [log]( https://travis-ci.org/conda-forge/conda-forge.github.io/jobs/130691832#L233-L244 ) and down below.

```
Prefix: /home/travis/conda/tmp_envs/f5607ad6e1b89da01f30
Running command: ['python', '/home/travis/build/conda-forge/conda-forge.github.io/scripts/update_teams.py', './feedstocks_repo/feedstocks']
Traceback (most recent call last):
  File "/home/travis/build/conda-forge/conda-forge.github.io/scripts/update_teams.py", line 91, in <module>
    contents = env.from_string(''.join(fh)).render()
  File "/home/travis/conda/tmp_envs/f5607ad6e1b89da01f30/lib/python3.5/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/travis/conda/tmp_envs/f5607ad6e1b89da01f30/lib/python3.5/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/travis/conda/tmp_envs/f5607ad6e1b89da01f30/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 2, in top-level template code
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'get'
CalledProcessError: Command '['python', '/home/travis/build/conda-forge/conda-forge.github.io/scripts/update_teams.py', './feedstocks_repo/feedstocks']' returned non-zero exit status 1
```

cc @pelson

### PR DESCRIPTION
**Pull request by ericdill**
This will probably take a few passes to get right.

Closes #139

### CODE REVIEW

**Comment by ericdill** [comment]
Ah looks like travis is a little backed up right now.  Let me see if I can figure out how to run this locally

**Comment by ericdill** [comment]
Nope. Need to have a github token that authorizes me for the conda-forge organization to run that script.  I'll need to wait until travis works out its backlog.

**Comment by jakirkham** [comment]
Looks like it did eventually run the CI. Thanks for adding this, @ericdill.

**Comment by jakirkham** [comment]
This seems like a totally reasonable fix to me. Let's give it a test drive. 🚗

If we decide we want something different, I'm open to swapping it out. It would just be nice to get the queue working again here. Hope that seems reasonable to others.

**Comment by jakirkham** [comment]
Unfortunately, testing here is pretty rough. So, the CI giving us a check mark doesn't mean as much as we would like. In the end, I find myself happier merging several PRs as I have seen @pelson do until we get this in some sort of working shape. Then we can go back and play with an ideal fix if it is needed.

**Comment by jakirkham** [comment]
Still failing, but a different error this time. It appears a path expected by `conda-build` is missing.

File: scripts/update_teams.py [hunk 1]
```diff
@@ -85,13 +75,9 @@ def create_team(org, name, description, repos):
     if not os.path.exists(recipe):
         print("The {} feedstock is recipe less".format(package_name))
         continue
-    env = jinja2.Environment(undefined=NullUndefined)
-
-    with open(recipe) as fh:
-        contents = env.from_string(''.join(fh)).render(os=os, environ=os.environ)
-    data = yaml.safe_load(contents)
+    meta = MetaData(recipe)
```
Comment (jakirkham) [inline scripts/update_teams.py]: I think we just want the directory this is in because I believe this is the `meta.yaml`.

Comment (ericdill) [inline scripts/update_teams.py]: Yep you are totally correct.  #143

**Comment by pelson** [comment]
Thanks for merging @jakirkham. I agree with the approach here. 👍

**Comment by jakirkham** [comment]
Thanks @pelson. Relieved to hear I wasn't getting way out of line here. 😌

**Pull request merged**